source/agora/node/Ledger.d

```diff
@@ -51,7 +52,7 @@ import agora.utils.PrettyPrinter;
 import std.algorithm;
 import std.algorithm.searching : maxElement;
 import std.conv : to;
-import std.exception;
+import std.exception : assumeUnique, assumeWontThrow;
```

Comment:
Need to do those changes because `std.exception` has `enforce` and so the names conflict.
Perhaps I should just pick another name ? But `enforce` seems like the best fit.

Reply:
`ensure` ?

Reply:
Good idea